lib/util/trip/drive_card.dart

```diff
@@ -1,75 +1,121 @@
+        return Theme.of(context).colorScheme.error;
+      } else if (_drive!.rides!.any((ride) => ride.status == RideStatus.pending)) {
+        return Theme.of(context).disabledColor;
+      } else {
+        return Theme.of(context).own().success;
+      }
+    }
+  }
+
+  BoxDecoration pickDecoration() {
+    if (_drive!.cancelled) {
+      return BoxDecoration(
+        color: Colors.grey,
+        borderRadius: cardBorder,
+        backgroundBlendMode: BlendMode.screen,
+      );
+    } else {
+      return BoxDecoration(
+        borderRadius: cardBorder,
+      );
+    }
+  }
+
   @override
   Widget build(BuildContext context) {
     return Card(
-      child: Semantics(
-        button: true,
-        tooltip: S.of(context).openDetails,
-        child: InkWell(
-          onTap: () => Navigator.of(context).push(
-            MaterialPageRoute(
-              builder: (context) => DriveDetailPage.fromDrive(trip),
+      color: pickBannerColor(),
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.circular(10),
+      ),
+      child: Wrap(
+        children: [
+          Container(
+            foregroundDecoration: pickDecoration(),
+            width: MediaQuery.of(context).size.width,
+            decoration: BoxDecoration(
+              color: Theme.of(context).cardColor,
+              borderRadius: cardBorder,
             ),
-          ),
-          child: Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              Padding(
-                padding: const EdgeInsets.all(12.0),
-                child: Text(localeManager.formatDate(trip.startTime)),
-              ),
-              const Divider(),
-              FixedTimeline(
-                theme: CustomTimelineTheme.of(context),
-                children: [
-                  TimelineTile(
-                    contents: Padding(
-                      padding: const EdgeInsets.all(32.0),
-                      child: Row(
-                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                        children: [
-                          Text('${localeManager.formatTime(trip.startTime)}  ${trip.start}'),
-                        ],
-                      ),
-                    ),
-                    node: const TimelineNode(
-                      indicator: CustomOutlinedDotIndicator(),
-                      endConnector: CustomSolidLineConnector(),
-                    ),
-                  ),
-                  TimelineTile(
-                    contents: Padding(
-                      padding: const EdgeInsets.all(32.0),
-                      child: Row(
-                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                        children: [
-                          Text('${localeManager.formatTime(trip.endTime)}  ${trip.end}'),
-                        ],
+            margin: const EdgeInsets.only(left: 10),
+            child: InkWell(
```

Comment:
this InkWell does not work anymore

Reply:
probably should be around the Wrap element?

Reply:
It InkWells the last ancestor normally. If I put before the Wrap it inkwells the Bannerbackgroundcard. Im still trying to figure out how to get it into the front layer.

Reply:
>  If I put before the Wrap it inkwells the Bannerbackgroundcard
How does that look? Can you send a picture?

Reply:
<img width="185" alt="Screenshot 2023-01-18 145539" src="https://user-images.githubusercontent.com/74068825/213189737-2d6348ca-13b0-47a4-bea5-28fd04922a96.png">

You don't really see it but the grey is different in this example

Reply:
I think if we want an Inkwell that is the best I can do at the momement :/

Reply:
Simon and I can try to fix that, maybe we will find something

Reply:
Done.